assets/vidstack_helper.js

```
@@ -1,199 +1,187 @@
-            if (consentSource.startsWith('youtube/') || consentSource.startsWith('vimeo/')) {
-                createConsentPlaceholder(videoContainer, player, consentText);
+    function initializeMediaPlayers() {
+        document.querySelectorAll('media-player[data-video-platform]').forEach(player => {
+            const platform = player.dataset.videoPlatform;
+            if (hasConsent(platform)) {
+                const videoId = player.dataset.videoId;
+                player.style.display = '';
+                player.setAttribute('src', `${platform}/${videoId}`);
+                const placeholder = player.previousElementSibling;
+                if (placeholder && placeholder.classList.contains('consent-placeholder')) {
+                    placeholder.style.display = 'none';
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)